### ISSUE TITLE
NOT BANNING

### ISSUE THREAD

**Issue by prT05** [Silenci0/SMAC#48]
One guy enter on my server and the name are changing to the name of players and he is using wallhack and aimbot and smac dont ban him

**Comment by borzaka** [comment]
Well, there is no perfect anti-cheat.
You could use the `smac_client.smx` plugin, it will kick players with name change spam.

**Comment by prT05** [comment]
Yes, but the insta ban, need to add a convar, or if they check if a player its name change spam, gets insta ban?

**Comment by prT05** [comment]
And one more quesiton, some people use random tags, Like AIM*Name, Bot*Name and they are always switch between 2 or 3 tags of group! This plugin will detect it like  acheat? Because this is a thing you can do on steam, and dont shoud ban im

**Comment by prT05** [comment]
like this https://www.youtube.com/watch?v=WS6C5BL2xkE

**Comment by borzaka** [comment]
There is no option to ban people for name changin spam I guess.
https://github.com/Silenci0/SMAC/wiki/Client-Protection

**Comment by prT05** [comment]
You said that they will get banned for: You could use the smac_client.smx plugin, it will kick players with name change spam.

**Comment by prT05** [comment]
So ban for spam change name

**Comment by borzaka** [comment]
Kick and ban are not the same. I said kick.
Changing name is not a cheat.
